sensor/AdblockPlugin.js

```diff
@@ -89,6 +89,20 @@ class AdblockPlugin extends Sensor {
       extensionManager.onGet("adblockConfig", async (msg, data) => {
         return this.getAdblockConfig();
       });
+      extensionManager.onSet("adblockFastMode", async (msg, data) => {
+        await rclient.hsetAsync("policy:system", "adblock_fastmode", JSON.stringify(data));
```

Comment:
use a constant variable for "adblock_fastmode" to avoid duplicate literal string scattered around